code/unclassified/TowerOfHanoi/TowerOfHanoi.scala

```diff
@@ -0,0 +1,24 @@
+import scala.collection.mutable.ArrayStack
```

Comment:
Add  this comment on the top of every code you submit  : -
/* Part of Cosmos by OpenGenus Foundation */     
👍